### PR TITLE
Update dependencies

### LINKED ISSUE #261: Update dependencies and build system
> This issue proposes a fix which does some updates.

### Background & Context

I thought about giving the tooling some attention again some time. Just updating things and add improvements I recently learned working on other projects.

Is that something we're interested in and doesn't come at a bad time?

### PR DESCRIPTION
**Pull request by tdeekens**
> This pull request update build dependencies.

### Background & Context

As proposed in #261 this pull request updates the build system's dependencies and adjusts e..g rollup API changes in the process.

Closes #261

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -5,16 +5,25 @@
     "format": "prettier --write --trailing-comma es5 --single-quote 'src/*.js'",
     "commit-amend-build": "scripts/commit-amend-build.sh",
     "prebuild": "rimraf dist/**",
-    "dev": "cross-env NODE_ENV=development BABEL_ENV=rollup rollup -w -c -o dist/purify.js",
+    "dev":
```
Comment (tdeekens) [inline package.json]: Nothing changed here. Just formatting.

File: test/karma.conf.js [hunk 2]
```diff
@@ -1,48 +1,48 @@
-const commonjs = require('rollup-plugin-commonjs');
-const includePaths = require('rollup-plugin-includepaths');
-const rollupConfig = require('../rollup.config.js');
-const customLaunchers = require('./karma.custom-launchers.config.js')
+const commonjs = require("rollup-plugin-commonjs");
```
Comment (tdeekens) [inline test/karma.conf.js]: Same here.

File: rollup.config.js [hunk 3]
```diff
@@ -1,37 +1,39 @@
...
 
 const config = {
-  entry: 'src/purify.js',
+  entry: "src/purify.js",
   external: [],
-  globals: {},
-  format: 'umd',
-  moduleName: 'DOMPurify',
-  sourceMap: true,
+  output: {
```
Comment (tdeekens) [inline rollup.config.js]: This is an API change from rollup around x.52.

**Comment by cure53** [comment]
Thanks :D

**Comment by tdeekens** [comment]
Sure. Always welcome.

**Pull request merged**